### PR TITLE
Update edge strings from FI

### PR DESCRIPTION
**Pull request by Colengms**

### CODE REVIEW

File: Extension/bin/messages/pt-br/messages.json [hunk 1]
```diff
@@ -3384,13 +3384,13 @@
...
-  "nenhum 'diagnóstico push #pragma' foi encontrado para corresponder a esse 'pop de diagnóstico'",
+  "nenhum 'diagnóstico push #pragma' foi encontrado para corresponder a esae 'pop de diagnóstico'",
   "o %sq não pode ser uma macro quando for usado em uma diretiva de importação ou módulo",
   "essa diretiva só pode aparecer no escopo de namespace global",
   "uma declaração 'exportar' só pode aparecer no escopo global ou de namespace",
   "%sq é analisado como um identificador em vez de uma palavra-chave porque os tokens que o seguem não correspondem aos de uma diretiva de pré-processador",
   "parece que esse é o início de uma diretiva de pré-processador, mas a falta de um '; ' seguido imediatamente por uma nova linha evita que",
   "parece ser uma diretiva de pré-processamento de módulos, mas essas diretivas não podem aparecer dentro de uma expansão de macro",
-  "uma diretriz 'módulo' não pode aparecer dentro do escopo de inclusão condicional (por exemplo, #if, #else, #elseif, etc.)",
+  "uma diretriz 'módulo' não pode aparecer dentro do escopo de inclusão condicional (por exemplo, #se, #outro, #outrose, etc.)",
```
Comment (Colengms) [inline Extension/bin/messages/pt-br/messages.json:3394]: @sean-mcmanus Do we already have a bug open against this string?

Comment (sean-mcmanus) [reply Extension/bin/messages/pt-br/messages.json:3394]: Yes, there's an open bug, but it was supposed to be fixed a long time ago.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
This change causes regressions and isn't needed -- there's one "real" change, but it should have no actual effect (the Italian one).

File: Extension/bin/messages/pt-br/messages.json (hunk 1 above)
Comment (Colengms) [reply Extension/bin/messages/pt-br/messages.json:3394]: @sean-mcmanus   It looks like these are all bad strings, and this PR can be abandoned.  Can you confirm?

Comment (sean-mcmanus) [reply Extension/bin/messages/pt-br/messages.json:3394]: Yes, see my comment: https://github.com/microsoft/vscode-cpptools/pull/8123#pullrequestreview-749750982

**Pull request closed without merging**